LoopKit/TemporaryScheduleOverride.swift

```diff
@@ -54,14 +54,27 @@ public struct TemporaryScheduleOverride: Hashable {
     public var startDate: Date
     public let enactTrigger: EnactTrigger
     public let syncIdentifier: UUID
+    
+    public var actualEnd: End = .natural
+    
+    public var actualEndDate: Date {
```

Comment:
The odd setter you mentioned above was actually being used to update *start* time as well, on instances of overrides only used for display; if there are multiple, it would need to adjust times on them to not draw them overlapping. I believe the new `scheduledInterval` is the right fix, and a setter is not needed here.